data/filetypes.Schema.conf

```diff
@@ -0,0 +1,66 @@
+# For complete documentation of this file, please see Geany's main documentation
+[styling]
+# Edit these in the colorscheme .conf file instead
+default=default
+commentline=comment_line
+number=number_1
+string=string_1
+character=character
+word=keyword_1
+triple=string_2
+tripledouble=string_2
+classname=type
+defname=function
+operator=operator
+identifier=identifier_1
+commentblock=comment
+stringeol=string_eol
+word2=keyword_2
+decorator=decorator
+
+[keywords]
+# all items must be in one line
+primary=attributetype objectclass ldapsyntax ditcontentrule attributeoptions objectidentifier
+# additional keywords, will be highlighted with style "word2"
+identifiers=NAME DESC OBSOLETE SUP EQUALITY ORDERING SUBSTR SYNTAX SINGLE-VALUE COLLECTIVE NO-USER-MODIFICATION ABSTRACT STRUCTURAL AUXILIARY MUST MAY
+
+[lexer_properties]
+fold.comment.python=1
+fold.quotes.python=1
+
+[settings]
```

Comment:
And you can do the same here, and then only override the parts you want, like this:

```conf
[settings=Python]
lexer_filetype=Python
# default extension used when saving files
extension=schema

# MIME type
#mime_type=text/x-python

# ...
```